src/init/starship.fish

```diff
@@ -23,7 +24,7 @@ function fish_right_prompt
     # Account for changes in variable name between v2.7 and v3.0
```

Comment:
You will also need to set the `STARSHIP_CMD_PIPESTATUS` env var in the `fist_right_prompt` function.

```suggestion
    set STARSHIP_CMD_PIPESTATUS $pipestatus
    # Account for changes in variable name between v2.7 and v3.0
```